apps/api/src/.env.test

```diff
@@ -44,7 +44,7 @@ S3_REGION=us-east-1
 GCS_BUCKET_NAME=novu-test
 AZURE_ACCOUNT_NAME=novu
 AZURE_ACCOUNT_KEY=123456
-AZURE_HOST_NAME=novu.blob.core.windows.net
+AZURE_HOST_NAME=https://novu.blob.core.windows.net
```

Comment:
This default config is invalid. In case it's set without protocol prefix, it will treat it as a pathname and end up with broken URL.